packages/backend/vitest.config.integration.ts

```diff
@@ -30,8 +30,13 @@ export default defineConfig({
       getPath('./test/ddb-global-setup.ts'),
       getPath('./test/redis-global-setup.ts'),
     ],
+    pool: 'threads',
+    poolOptions: {
+      threads: {
+        singleThread: true,
+      },
+    },
```

Comment:
noob question: actly why do we run the tests in a single thread ah, is this for correctness or speed ah

Reply:
it's only for integration tests, because we purge the db after each test, we dont want to purge in the middle of another running test